librdkafka.sh

```diff
@@ -3,20 +3,21 @@ version: "%(tag_basename)s"
 tag: v2.3.0
 requires:
   - "GCC-Toolchain:(?!osx)"
+  - curl
```

Comment:
why do you require curl if you disable it later?

Reply:
good point, fixed